src/views/home.js

```diff
@@ -1,18 +1,94 @@
-import {Text, View, Button, StyleSheet} from 'react-native';
-import React from 'react';
+import {
+  Text,
+  View,
+  StyleSheet,
+  FlatList,
+  ScrollView,
+  TouchableOpacity,
+} from 'react-native';
+import React, {useEffect} from 'react';
+import {TodoItem} from '../components/TodoItem';
+import {incrementDateByDays} from '../utils/utils';
+import Icon from 'react-native-ionicons';
+import ScreenTitle from '../components/screenTitle';
+
+export const Home = ({navigation, route}) => {
+  const DATA = [];
+  var today = new Date().getTime();
+  for (var i = 0; i < 100; i++) {
+    DATA.push({
+      id: i.toString(),
+      title: 'Title ' + i,
+      description: 'Description ' + i,
```

Comment:
É melhor usar interpolação de strings em vez de + porque o JS pode faze somas com strings

```suggestion
      title: `Title ${i}`,
      description: `Description ${i}`,
```